packages/react/src/components/TextArea/TextArea.stories.js

```diff
@@ -147,21 +113,60 @@ Default.args = {
   rows: 4,
   warn: false,
   warnText: 'This is a warning message.',
+  cols: 50,
+};
+export default {
+  title: 'Components/TextArea',
+  component: TextArea,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  subcomponents: {
+    TextAreaSkeleton,
+  },
+  argTypes: {
+    ...sharedArgTypes,
+    light: {
+      table: {
+        disable: true,
+      },
+    },
+    slug: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  args: {
+    ...sharedArgs,
+  },
+};
+
+export const Default = (args) => {
+  return <TextArea {...args} id="text-area-1" />;
 };
 
-export const _WithLayer = () => (
+Default.args = {
+  enableCounter: true,
+};
+
+export const _WithLayer = (args) => (
   <WithLayer>
     {(layer) => (
       <TextArea
         labelText="Text Area label"
         helperText="Optional helper text"
         rows={4}
         id={`text-area-${layer}`}
+        {...args}
       />
     )}
   </WithLayer>
 );
 
+_WithLayer.args = { ...sharedArgs, helperText: 'Optional helper text' };
```

Comment:
```suggestion
_WithLayer.args = { helperText: 'Optional helper text' };
```

This goes along with the comment made above about `sharedArgs`. Putting `args` inside of the `export default` block will apply the args to every story, so there will be no need for this variable.